docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,188 @@
+.. _use-trained-models:
+
+Using Trained Checkpoints
+=================================
+
+Determined provides APIs for downloading trained checkpoints and loading them
+into memory in a python process.
+
+Querying Checkpoints
+--------------------
+
+The :class:`ExperimentReference <determined.experimental.ExperimentReference>`
+class contains the ``top_checkpoint`` method. Without arguments the method will
+check the experiment configuration searcher field for the ``metric`` and
+``smaller_is_better`` values. These values are used to sort the experiments
+checkpoints by validation performance. For example the following searcher
+configuration will result in `ExperimentReference.top_checkpoint` returning the
+checkpoint with the lowest "loss" metric.
+
+.. code:: yaml
+
+   searcher:
+     metric: "loss"
+     smaller_is_better: true
+
+The following snippet will return an instance of :class:`Checkpoint
+<determined.experimental.Checkpoint>` representing the checkpoint from an
+experiment with the best validation metric.
```

Comment:
let's explain here that this should be in any python script after a checkpoint has been made.